File: src/main/java/a6n_test_002/Solution.java
import java.io.*;
import java.math.*;
import java.security.*;
import java.text.*;
import java.util.*;
import java.util.concurrent.*;
import java.util.function.*;
import java.util.regex.*;
import java.util.stream.*;
import static java.util.stream.Collectors.joining;
import static java.util.stream.Collectors.toList;

class Result {

    /*
     * Complete the 'getUsernames' function below.
     *
     * The function is expected to return a STRING_ARRAY. The function accepts
     * INTEGER threshold as parameter.
     *
     * URL for cut and paste
     * https://jsonmock.hackerrank.com/api/article_users?page=<pageNumber>
     */

    public static List<String> getUsernames(int threshold) {

    }

}

public class Solution {
    public static void main(String[] args) throws IOException {
        BufferedReader bufferedReader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(System.getenv("OUTPUT_PATH")));

        int threshold = Integer.parseInt(bufferedReader.readLine().trim());

        List<String> result = Result.getUsernames(threshold);

        bufferedWriter.write(result.stream().collect(joining("\n")) + "\n");

        bufferedReader.close();
        bufferedWriter.close();
    }
}

File: src/main/java/a6n_test_002/solution.lua
--
-- Complete the 'getUsernames' function below.
--
-- The function is expected to return a STRING_ARRAY.
-- The function accepts INTEGER threshold as parameter.

-- URL for cut and paste
-- https://jsonmock.hackerrank.com/api/article_users?page=<pageNumber>
--

function getUsernames(threshold)

end

local fptr = io.open(os.getenv("OUTPUT_PATH"), "w")

local threshold = io.stdin:read("*n", "*l")

local result = getUsernames(threshold)

fptr:write(table.concat(result, "\n"))

fptr:write("\n")

fptr:close()
